app/jobs/webhook_deliveries/deliver_job.rb

```diff
@@ -4,17 +4,23 @@ class DeliverJob < ApplicationJob
 
     def perform(event_type, course, resource)
       webhook_endpoint = course.webhook_endpoint
+
       payload = {
         data: data(event_type, resource),
         event: event_type
       }
+
       uri = URI.parse(webhook_endpoint.webhook_url)
+
       request = Net::HTTP::Post.new(uri.request_uri)
       request['Content-Type'] = 'application/json'
       request.body = payload.to_json
+      request['Authorization'] = "PF-HMAC-SHA256 #{hmac(webhook_endpoint.hmac_key, request.body)}"
```

Comment:
do we need that empty space after `PF-HMAC-SHA256`

Reply:
@bodhish Yes. The `Authorization` header has the format `Authorization: <type> <credentials>`

This type is obviously non-standard, but I've kept it similar to a de facto standard - Amazon's `AWS4-HMAC-SHA256`. They use the header differently, but I've kept the type naming similar.